french/dessine/porte.md

```diff
@@ -3,5 +3,6 @@ La porte qui m'a attire le plus est la porte dessinee avec du miel. Elle brillai
 Donc je decide de
 
 [Manger un peu de miel](miel/manger-du-miel.md)
+[Essayer d'ouvrir la porte faite de chocolat](chocolat/essayer-ouvrir-chocolat.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.